baseline-error-prone/src/test/java/com/palantir/baseline/errorprone/CatchSpecificityTest.java

```diff
@@ -560,7 +560,7 @@ void typeParameterExceptionToRuntimeException() {
                         "  <T extends RuntimeException> void f(ThrowingRunnable<T> in) {",
                         "    try {",
                         "        in.run();",
-                        "    } catch (Exception e) {",
+                        "    } catch (RuntimeException e) {",
```

Comment:
this looked like a bug to me - in this test we know we throw T which in this scope is guaranteed to be RuntimeException (while unbounded T is Exceptio)

Reply:
:+1:

Reply:
So it turns out this is a bit more subtle -- using a java9+ compiler to target java8 this works as expected, however the java 8 compiler cannot compile this output!
Not recommending that we revert since everything will be 11+ soon, and it's rare enough that I'd rather suppress.